sonar-batch/src/main/java/org/sonar/batch/repository/DefaultProjectRepositoriesLoader.java

```diff
@@ -58,7 +54,6 @@ public ProjectRepositories load(String projectKeyWithBranch, @Nullable String so
     url += "&preview=" + analysisMode.isIssues();
 
     ProjectRepositories projectRepositories = load(url, fromCache);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "projectRepositories". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)